keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -51,16 +48,18 @@ Consistent reads may be served from cache so long as:
 - The data in the watch cache no older than the latest "revision" just from etcd
 
 etcd watches support "progress events", which provide an updated revision and a
-guarantee that all future watch events will be newer than the that revision.  If
-an etcd watch is configured with `WithProgressNotify` enabled, etcd
-automatically sends progress events at a regular interval. The "progress events"
-allow a etcd watcher to know how up-to-date the watch stream is relative a
-particular revision.
+guarantee that all future watch events will be newer than the that revision. 
+Etcd client can request a progress notification from server. The progress 
```

Comment:
I can't find it now, but I have a feeling that I've seen somewhere that "client requested progress" is delivered not only to my watch, but it triggers delivering progress-notification for *all* watchers.

I hope I'm wrong? Can you point me to the right documentation (I can't find something useful here).


[If this wouldn't be true, that has a potentially serious scalability implication.]

Reply:
Progress is requested on client, so all watches opened with this client will get the notification.